public/app/features/dashboard-scene/scene/LibraryVizPanel.tsx

```diff
@@ -78,9 +89,17 @@ export class LibraryVizPanel extends SceneObjectBase<LibraryVizPanelState> {
 function getLoadingPanel(title: string) {
   return new VizPanel({
     title,
+    $data: new SceneDataTransformer({ transformations: [], $data: new SceneQueryRunner({ queries: [] }) }),
```

Comment:
Same deal here as with the links. We're expecting this to exist on the panel, so we need a dummy data provider until the library panel loads.

Reply:
@torkelo it's _was_ needed here, but I forgot to remove it after other changes. https://github.com/grafana/grafana/blob/26588eed3d9075ba48bd41032057edfdf70ee16e/public/app/features/dashboard-scene/panel-edit/PanelDataPane/PanelDataPane.tsx#L102

Panels that are not data panels do not render the `PanelDataPane`. But the dummy loading panel is a data panel so this was still renderd.

But I later changed this so it does not render the edit view before the panel is loaded. https://github.com/grafana/grafana/blob/26588eed3d9075ba48bd41032057edfdf70ee16e/public/app/features/dashboard-scene/scene/DashboardSceneUrlSync.ts#L116-L120

This return early now.

I'll remove the queryrunner from the loading panel.